src/components/subscription-choices/subscription-choices.stories.mdx

```diff
@@ -0,0 +1,179 @@
+| `notifications_btn_class`                | string | `''`                                    | Appends a CSS class(es) to the "Get notifications" button                                                       |
+| `notifications_btn_initial_label`        | string | `'Notifications have been turned off.'` | See [Button Swap `initial_label`](?path=/docs/components-button-swap--default-story#template-properties)        |
+| `notifications_btn_initial_visual_label` | string | `'Get notifications'`                   | See [Button Swap `initial_visual_label`](?path=/docs/components-button-swap--default-story#template-properties) |
+| `notifications_btn_swapped_label`        | string | `'Notifications have been turned on.'`  | See [Button Swap `swapped_label`](?path=/docs/components-button-swap--default-story#template-properties)        |
+| `notifications_btn_swapped_visual_label` | string | `'Turn off notifications'`              | See [Button Swap `swapped_visual_label`](?path=/docs/components-button-swap--default-story#template-properties) |
+| `weekly_digests_btn_class`               | string | `''`                                    | Appends a CSS class(es) to the "Weekly Digests" button                                                          |
+| `weekly_digests_btn_label`               | string | `'Get Weekly Digests'`                  | The "Weekly Digests" button label                                                                               |
+| `weekly_digests_btn_icon`                | string | `'envelope' `                           | The "Weekly Digests" button [icon](?path=/docs/design-icons--page)                                              |
+| `weekly_digests_heading`                 | string | `'Get Weekly Digests'`                  | The subscription form heading                                                                                   |
+| `email_input_label`                      | string | `'Email'`                               | The email input label                                                                                           |
+| `email_input_placeholder`                | string | `'Your Email Address'`                  | The email input placeholder text                                                                                |
+| `submit_btn_label`                       | string | `'Subscribe'`                           | The subscription form submit button label                                                                       |
+
+## JavaScript
+
+The Subscription Choices component UX can be progressively enhanced by adding
+a 1 second form hide delay when `:focus` is moved away from the Subscription
+Choices component elements.
+
+### Syntax
+
+```js
+initSubscriptionChoices(subscriptionChoicesEl);
```

Comment:
Does the name of the JS stuff need to be updated to match the new component name?